VazhovIA/lab1/calculator/main.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+int main()
+{
+    float left, right;
+    char sign;
+    cin >> left >> sign >> right;
+
+    switch (sign)
+    {
+    case '+':
+        cout << left + right << endl;
```

Comment:
Store result of arithmetic operation in separated variable and print before app closing